buffer/src/main/java/io/netty/buffer/PoolChunk.java

```diff
@@ -245,18 +239,18 @@ private void insertAvailRun(int runOffset, int pages, Long handle) {
         }
     }
 
-    private void insertAvailRun0(int runOffset, Long handle) {
-        Long pre = runsAvailMap.put(runOffset, handle);
-        assert pre == null;
+    private void insertAvailRun0(int runOffset, long handle) {
+        long pre = runsAvailMap.put(runOffset, handle);
+        assert pre == 0;
```

Comment:
what is the reason for returning `0` and not `-1` here ? IMHO `-1` would be what I expect for a non existing key as handles will always be positive.